core/src/ast_builder/expr/numeric.rs

```diff
@@ -0,0 +1,127 @@
+use {
+    crate::{
+        ast::AstLiteral,
+        ast_builder::AstBuilderError,
+        result::{Error, Result},
+    },
+    bigdecimal::BigDecimal,
+    std::{borrow::Cow, str::FromStr},
+};
+
+#[derive(Clone)]
+pub enum NumericNode<'a> {
+    I8(i8),
+    I16(i16),
+    I32(i32),
+    I64(i64),
```

Comment:
Is there a reason the 128 size is missing?

Reply:
For now, it is just because `bigdecimal` crate does not support from 128 size to bigdecimal